parser_library/test/lsp/lsp_context_ord_sym_test.cpp

```diff
@@ -214,3 +215,19 @@ B DS F
     EXPECT_EQ(a_pos, (std::vector<position> { { 3, 13 }, { 6, 15 }, { 9, 0 } }));
     EXPECT_EQ(b_pos, (std::vector<position> { { 7, 22 }, { 10, 0 } }));
 }
+
+TEST(hover, using)
+{
+    std::string input = R"(
+    USING *,15
```

Comment:
Not sure what should happen with invalid register number:

![image](https://github.com/BroadcomMFD-DevOps/che-che4z-lsp-for-hlasm/assets/99467904/5069ece3-8cdb-45cc-bb07-6a68db181b0e)

Reply:
This is a bit of hard problem, because the active usings depend on previous usings, so an error anywhere in the history makes the whole chain invalid. In this first version, I would leave it as it is (garbage in, garbage out approach), maybe in the future an "invalid" or "corrupted" flag could be carried between the states as they are being evaluated, which we could then show in the hover box to let the user know which usings are currently unreliable due to errors.